src/conn/conn_ckpt.c

```diff
@@ -26,8 +26,7 @@ __ckpt_server_config(WT_SESSION_IMPL *session, const char **cfg, bool *startp)
     conn = S2C(session);
 
     WT_RET(__wt_config_gets(session, cfg, "checkpoint.precise", &cval));
-    /* Disaggregated storage only supports precise checkpoint. */
```

Comment:
I think we should have a more detailed comment here explaining the situation. It's still true long-term that disagg will only use precise checkpoint, but not automatically setting that is a workaround until mongo advances the stable timestamp and we should probably mention that.